static/src/javascripts/projects/commercial/modules/article-body-adverts.js

```diff
@@ -68,10 +74,10 @@ const insertAdAtPara = (
 };
 
 // Add new ads while there is still space
-const addArticleAds = (count: number, rules: Object): Promise<number> => {
+const addArticleAds = (rules: Object, count: ?number): Promise<number> => {
     const insertInlineAds = (paras: HTMLElement[]): Promise<number> => {
         const slots: Array<Promise<void>> = paras
-            .slice(0, Math.min(paras.length, count))
+            .slice(0, count ? Math.min(paras.length, count) : paras.length)
```

Comment:
I'd be explicit here about whether you're checking `count === undefined` or `count != 0`